docs/Laravel.md

```diff
@@ -99,7 +99,9 @@ hooks:
 Since we are writing the config cache to disk, all the paths in the config file will be resolved. Since those paths do not exist on our machine (they exist on the lambda environment only) we will have fake paths in the cached config. This is a problem in `config/views.php` because `realpath()` is used. We need to remove the `realpath()` call:
 
 ```diff
--    'compiled' => realpath(storage_path('framework/views')),
+-    'compiled' => env(
+-    'VIEW_COMPILED_PATH',
+-    realpath(storage_path('framework/views'))
 +    'compiled' => storage_path('framework/views'),
```

Comment:
Same here we can now override `VIEW_COMPILED_PATH` to avoid modifying this line manually! This should be done in `.env.production` I guess.